DroneCones/DroneConesApp/templates/DroneConesApp/landing.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'DroneConesApp/misc/base.html' %}
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="{% static 'DroneConesApp/mainStyle.css' %}">
+    <script src="{% static 'DroneConesApp/final.js' %}" defer></script>
+ <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+</head>
+{% block content %}
```

Comment:
Do you think you could explain to me the `block content` usage?  I thought extending `base.html` would put the entire file in the body of `base.html` therefore no head is needed in any file that extends `base.html`?

Reply:
^^

Reply:
I can fix this. I didn't necessarily do anything in the landing page, but Porter did some linking stuff to the sign in pages on this branch. I think that this is just a result.